src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -352,43 +137,28 @@ public void EmitBindingCompleteMessage()
             testSubject.InternalState.BindingOperationSucceeded.Should().BeTrue($"Initial state of {nameof(BindingProcessImpl.InternalState.BindingOperationSucceeded)} should be true");
         }
 
-        [TestMethod]
-        public void SaveRuleConfiguration_CallsSolutionBindingOperation()
```

Comment:
The `SaveRuleConfiguration` method is now a no-op and will be deleted later, so I just removed this test.